socketio.js

```diff
@@ -303,7 +303,7 @@ function can_subscribe_list(args) {
 				}
 				return false;
 			} else if (res.status == 200) {
-				args?.callback(err, res);
+				args.callback(err, res);
```

Comment:
```suggestion
				args.callback && args.callback(err, res);
```

This is not equivalent. 

Also root cause is most likely your actual socketio process isn't using node14 which does support this. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining